capstone/src/main/java/com/google/step/clients/ScoreClient.java

```diff
@@ -86,10 +88,13 @@ private double calculateRestaurantAveragePageViews(RestaurantPageViews restauran
         List<WeeklyPageView> pageViews = restaurant.getPageViews();
 
         double sum = 0;
+
+        // This can't be zero because the instance of a RestaurantPageViews only exists if a WeeklyPageView
```

Comment:
Is the Cloud Scheduler job's execution contingent on the fact that there are page views stored in Datastore? I don't quite understand how this is the case.

Reply:
No, its because if this method gets called, it is certain that there will be page views in the system, and if there are pageViews in the system then the count can't be 0.